server/athenian/api/openapi/openapi.yaml

```
@@ -1435,3 +1513,327 @@ components:
+            hash: 79f0bccb856b35ef08ec93da2ff5585da4d2e50e
+            author:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+            committer:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+              message: 'Bump version: 0.0.42 → 0.0.43'
+              size_added: 1
+              size_removed: 1
+              files_changed: 1
+          - repository: github.com/athenianco/athenian-api
+            hash: 79f0bccb856b35ef08ec93da2ff5585da4d2e50e
+            author:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+            committer:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+              message: 'Bump version: 0.0.42 → 0.0.43'
+              size_added: 1
+              size_removed: 1
+              files_changed: 1
+        - date: 2000-01-23
+          commits:
+          - repository: github.com/athenianco/athenian-api
+            hash: 79f0bccb856b35ef08ec93da2ff5585da4d2e50e
+            author:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+            committer:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+              message: 'Bump version: 0.0.42 → 0.0.43'
+              size_added: 1
+              size_removed: 1
+              files_changed: 1
+          - repository: github.com/athenianco/athenian-api
+            hash: 79f0bccb856b35ef08ec93da2ff5585da4d2e50e
+            author:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+            committer:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+              message: 'Bump version: 0.0.42 → 0.0.43'
+              size_added: 1
+              size_removed: 1
+              files_changed: 1
+      properties:
+        data:
+          items:
+            $ref: '#/components/schemas/CommitGroup'
+          type: array
+        include:
+          $ref: '#/components/schemas/IncludedNativeUsers'
+      required:
+      - data
+      - include
+      type: object
+    CommitGroup:
+      description: List of commits that were committed starting from a specific time
```

In order to make it more flexible, I'd not mention the interval not the granularity, which is something about the query using this entity, which can use an interval, or not.
```suggestion
      description: List of commits
```